pkgs/misc/blesh/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchgit }:
+stdenv.mkDerivation rec {
+  pname = "blesh";
+  version = "0.3.3"; 
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "398e404d624ef6365d9175882e24f6e180d8b6b2";
+    sha256 = "09i6a45lcd8gb0397hcffwsc5fqam8f7wk2yw9qgjjrb7i9n1235";
+  };
+  configurePhase = ''
+    mkdir contrib/.git 
+  '';
+  installPhase = ''
+    mkdir -p $out/share/doc
+    mv out/doc $out/share/doc/blesh
+    mv out $out/share/blesh
+  '';
+
```

Comment:
I would find it useful to add the `ble.sh` shell script to PATH, if that is still acceptable even though it is not an executable. This would do it:

```
preFixup = ''
  mkdir $out/bin
  cat >$out/bin/ble.sh <<EOF
  source $out/ble.sh
  EOF
'';
```